jose-util/README.md

```diff
@@ -0,0 +1,54 @@
+# JOSE CLI
+
+The `jose-util` command line utility allows for encryption, decryption, 
+signing and verification of JWE/JWS message on the command line. It's main
+purpose is to facilitate dealing with JWE/JWS messages when testing or
+debugging.
+
+## Usage
+
+The utility has various subdommands. Valid commands are `encrypt`, `decrypt`,
```

Comment:
typo: s/subdommands/subcommands/

Or, just say "The utility includes the subcommands `encrypt`, `decrypt`, `sign`, `verify`, and `expand`."